### DISCUSSION TITLE
Curly/colored underlines not working on windows

### DISCUSSION THREAD

**Discussion by chtenb** [helix-editor/helix#10245] [Troubleshooting]
### Summary

Enabling a test theme with many curly underlines yields this result
cmd.exe: ![image](https://github.com/helix-editor/helix/assets/1030961/cefb9fe4-1af2-49e2-9bd3-42cda3ba8ce2)
wezterm: 
![image](https://github.com/helix-editor/helix/assets/1030961/333752c2-8ead-40a9-a40c-15054997244a)

However, both terminal emulators do support curly underlines.
cmd.exe: 
![image](https://github.com/helix-editor/helix/assets/1030961/20e79251-f7cc-47a3-972f-f271db68743a)

wezterm:
![image](https://github.com/helix-editor/helix/assets/1030961/4428c178-afe9-45e7-9389-398de996f677)

### Reproduction Steps

 

I tried this:

1. Configure a theme with curly underlines, `"ui.text" = { underline = { color = "light-green", style = "curl" } }`
2. Open a text file in `hx` and observe

I expected this to happen:
Green curly underlines

Instead, this happened:
straight colorless underlines

### Helix log

No relevant log lines

### Platform

Windows 11

### Terminal Emulator

Latest wezterm and cmd.exe

### Installation Method

source

### Helix Version

24.03-21-g55c12d90

**Comment by pascalkuthe** [comment]
Detecting support on windows automatically is hard since there is no terminfo.

You.nerd to manually enable the config option

**Reply by kirawi** [reply]
Edit: Ah nevermind, it never goes through the branch if there's no terminfo to begin with.

Shouldn't Wezterm have undercurls regardless because of https://github.com/helix-editor/helix/blob/97afd67fca0b505600e5fbba7ca59f06a4eec3ff/helix-tui/src/backend/crossterm.rs#L63-L76?
